src/transformers/models/vilt/modeling_vilt.py

```diff
@@ -933,6 +933,9 @@ def forward(
         >>> import requests
         >>> from PIL import Image
         >>> import re
+        >>> import torch
+
+        >>> device = torch.device("cuda:0" if torch.cuda.is_available() else "cpu")
```

Comment:
It will fail on GPU

Reply:
You are right! I added the line `>>> device = torch.device("cuda:0" if torch.cuda.is_available() else "cpu")` just because the line `input_ids = torch.tensor(encoded.input_ids).to(device)` was already there, and the example fails because `device` was not defined. I can do:

- change the line to `input_ids = torch.tensor(encoded.input_ids)`
- remove the added line `>>> device = torch.device("cuda:0" if torch.cuda.is_available() else "cpu")` 

Is this OK for you?

Reply:
Yes that's OK!